quick-start.md

```diff
@@ -16,7 +16,7 @@ TinyMCE 5.0 is perfect for developers who want to see how the new version of Tin
 Include this line of code in the `<head>` of the HTML page and link to the tinymce.min.js source file:
 
 ```html
-<script src="{{ site.cdnurl }}?apiKey=your_API_key"></script>
+<script src="{{ site.cdnurl }}" referrerpolicy="origin"></script>
```

Comment:
We don't explain no-api-key here.

Reply:
added: http://docs-v5-feature-tcloud-2861-2.staging.tiny.cloud/docs/quick-start/